src/main/ank/1.4-kleisli-categories.md

```diff
@@ -0,0 +1,69 @@
+```Haskell
+type Writer a = (a, String)
+```
+```kotlin:ank:silent
+typealias Writer<T> = Pair<T, String>
```

Comment:
In Arrow we don't use `T` but `A`, `B`... for mono types and `F`, `G`... for poly ones.
```suggestion
import arrow.core.Tuple2

typealias Writer<A> = Tuple2<A, String>
```